html/index.html

```diff
@@ -3,19 +3,21 @@
 <head>
   <meta charset="UTF-8">
   <title>Adam Dow</title>
-  <link href='css/less/styles.css' rel='stylesheet'>
   <link href="https://fonts.googleapis.com/css?family=Raleway:700|Montserrat&display=swap" rel="stylesheet">
```

Comment:
you need to add this for responsive

<meta name="viewport" content="width=device-width, initial-scale=1.0">
look it up

Reply:
`<meta name="viewport" content="width=device-width, initial-scale=1.0">`